### PR TITLE
Disable unsupported/failing raptor tests

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java [hunk 1]
```diff
@@ -49,7 +49,14 @@ public void testCreateArrayTable()
         assertQueryTrue("DROP TABLE array_test");
     }
 
-    @Test
+    @Test(enabled = false)
```
Comment (losipiuk) [inline presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java:52]: why not @Ignore?

File: presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java [hunk 2]
```diff
@@ -49,7 +49,14 @@ public void testCreateArrayTable()
...
+    @Override
+    public void testCreateTableAsSelect()
```
Comment (losipiuk) [inline presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java:54]: why did you add new test and ignored it?

File: presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java (hunk 1 above)
Comment (sopel39) [inline presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java:52]: no such annotation in testng

File: presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java (hunk 2 above)
Comment (sopel39) [inline presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java:54]: ditto

**Comment by kokosing** [comment]
I would not merge this, instead I would work to fix this
